### ISSUE TITLE
Clone tool

### ISSUE THREAD

**Issue by AlexSlivkin97** [bgrabitmap/lazpaint#271]
When cloning, the first point disappears and it is not visible that you are copying. With a large number of elements, it lags

**Comment by circular17** [comment]
Yes, that was intended to disappear. Though I reckon you may want to continue to see the source point. I note that as a possible enhancement.

About the lag, am not sure of the context.

**Comment by sganz** [comment]
Was just about to put a feature / enhance request for a similar request. 

I just started trying to use Lazpaint for image editing, and first thing I tried was the clone tool. It would be nice to see the Source Point as you clone, saves an undo when you don't remember where it's at while cleaning up an image. Also possible options for color of the Tool and Mark to make the tool more visible, on B&W images it's sometimes hard to see where it's at and locate it. 

As an FYI I tried 11660x8904 JPEG as well as same size TIFF file, seemed to be OK with the huge images I scanned. 

Playing and learning 😄

**Comment by circular17** [comment]
Agreed, so the idea would be to add:
- an option to always keep the source point visible
- an option to choose the color of the source point indicator

Regards
